windows/src/desktop/kmshell/main/initprog.pas

```diff
@@ -249,6 +261,7 @@ function Init(var FMode: TKMShellMode; KeyboardFileNames: TStrings; var FSilent,
       else if s = '-h'   then FMode := fmHelp
       else if s = '-t'   then FMode := fmTextEditor
       else if s = '-ouc' then FMode := fmOnlineUpdateCheck
+      else if s = '-buc' then FMode := fmBackgroundUpdateCheck
```

Comment:
I don't see where we deal with `fmBackgroundUpdateCheck`. I would have expected some new code similar to where we deal with `fmInstallBackground`.